examples/deepcopy-gen/generators/deepcopy.go

```diff
@@ -761,7 +761,12 @@ func (g *genDeepCopy) doMap(t *types.Type, sw *generator.SnippetWriter) {
 			} else if uet.Kind == types.Pointer {
 				sw.Do("if val==nil { (*out)[key]=nil } else {\n", nil)
 				sw.Do("(*out)[key] = new($.Elem|raw$)\n", uet)
-				sw.Do("val.DeepCopyInto((*out)[key])\n", nil)
+				if uet.Elem.Kind == types.Builtin {
+					sw.Do("x := *val\n", nil)
```

Comment:
Won't this kind of trash Go's concept of heap and stack?  I know Go is smart enough to move things but ISTR that the general guidance for perf was to avoid t if you can.  In this case, maybe we should use `new` ?

Reply:
Also, we used `new` on L763, can we just assign into that value here

Reply:
fixed